### PR TITLE
5:Fixed back button disappearance

### PR DESCRIPTION
**Pull request by EMG114**
Fix back button disappearance when pushed. The back button in the city selection used to disappear when tapping on the label.

### CODE REVIEW

File: ParkenDD/LotlistView/LotlistViewController.swift [hunk 1]
```diff
@@ -76,6 +76,11 @@ class LotlistViewController: UITableViewController, UIViewControllerPreviewingDe
 
 		refreshControl?.endRefreshing()
 	}
+	
+	override func viewWillDisappear(_ animated: Bool) {
```
Comment (olivaresf) [inline ParkenDD/LotlistView/LotlistViewController.swift]: This still doesn't fix the issue for me. However, you're on the right track. Try moving this to `SettingsViewController`'s `viewDidLoad`. Make sure that the title is the localized string.

**Review by olivaresf — CHANGES_REQUESTED** [review]

File: ParkenDD/LotlistView/LotlistViewController.swift (hunk 1 above)
Comment (EMG114) [reply ParkenDD/LotlistView/LotlistViewController.swift]: It seems to me that you would need it in both SettingsViewController and LotlistViewController. There seems to be some bug with the back navigation button when you go from LotlistVC to CityListVC and from SettingsVC to CityListVC. Just putting it only on SettingsVC does solve the problem. Any VC that pushes to CItyListVC has this same problem. So as a solution I am proposing having it in both VCs.

Comment (EMG114) [reply ParkenDD/LotlistView/LotlistViewController.swift]: Provided a commit with a possible fix. Read comment above. Let me know.

**Review by olivaresf — APPROVED** [review]
Just tested it and you're right. This looks good.

**Pull request merged**